components/Frame/Popover/Nav.js

```diff
@@ -197,6 +197,28 @@ const Nav = ({ me, url, closeHandler, children, t, inNativeApp, inNativeIOSApp,
               <br />
             </Fragment>
           )}
+          {!!me && (
```

Comment:
Currently quite unbalanced:
<img width="1209" alt="screen shot 2018-09-02 at 14 33 42" src="https://user-images.githubusercontent.com/410211/44956011-38cfe200-aebd-11e8-9298-5ff0248bedfa.png">

Consider moving to the left side underneath `Konto`. And use better label: `Teilen` is too generic. Maybe `Zugriff teilen`.

Reply:
better label idea: `Republik teilen`